### ISSUE TITLE
focus lost when switching windows with alt+tab

### ISSUE THREAD

**Issue by RDMurray** [nvaccess/nvda#20822]
### Brief summary

This appears to be a Windows bug because Narrator has similar behaviour.  When switching from Windows Terminal to other applications with alt+tab or Win+Tab, NVDA doesn't recognise the newly focused application. 

### Steps to reproduce

1. Open windows terminal and another application, e.g. notepad.
2. focus the terminal
3. use alt+tab to switch to notepad.
4. press NVDA+T, notice that it says "Task Switching" for alt+tab or "Explorer" for Win+tab.
5. If it isn't reproduced the first time, try alt+tab again. It happens 100% of the time here when switching from terminal to Firefox.

### Actual behavior

- NVDA+T
- Windows PowerShell  terminal
- alt+tab
- NVDA Speech Viewer  2 of 4
- alt+tab
- Issues · nvaccess/nvda — Mozilla Firefox  3 of 4
- let go of ALT
NVDA+t
Task Switching

### Expected behavior

The actual focused window should be recognised by NVDA.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/32036994/nvda.log)

### NVDA type

installed copy

### NVDA version

2026.3beta1 (2026.3.0.57758)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

I think it has been a while, but suspect a Windows bug so probably any version.

### Windows version

Version	10.0.26200 Build 26200

### Name and version of other software in use when reproducing the issue

Windows Terminal, Microsoft Corporation, 1.24.11911.0

### Other information about your system

Win11 Pro on Lenovo X13 gen5

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have run the System Accessibility Repair Tool and the issue still occurs

**Comment by tareh7z** [comment]
Hi @RDMurray!

I can occasionally reproduce this myself, but only under certain conditions — for example, after I've been working in a terminal that was heavily flooded with output.

Interestingly, simply restarting NVDA fixes the problem for me. So while this may indeed be a Windows issue, the fact that restarting NVDA alone is enough to recover from it makes me wonder whether there might still be something on the NVDA side that gets into a bad state after interacting with Windows Terminal.

Thanks.
